src/sparseml/pytorch/image_classification/README_image_classification.md

```diff
@@ -0,0 +1,253 @@
+                                  using a SparseZoo recipe, can also provide
+                                  'zoo' to load the base weights associated
+                                  with that recipe. Additionally, can also
+                                  provide a SparseZoo model stub to load model
+                                  weights from SparseZoo
+  ...
+```
+
+To learn about the sparsification in more detail, refer to [SparseML docs](https://docs.neuralmagic.com/sparseml/)
+
+## Once the Training is Done...
+
+The artifacts of the training process are saved to the `--save-dir` under `--model-tag`.
+Once the script terminates, you should find everything required to deploy or further modify the model,
+including the recipe (with the full description of the sparsification attributes), checkpoint files (saved in the appropriate framework format), etc.
+
+### Exporting the Sparse Model to ONNX
+
+The [DeepSparse] Engine uses the ONNX format to load neural networks and 
+deliver breakthrough performance for CPUs by leveraging the sparsity and 
+quantization within a network.
+
+The SparseML installation provides a `sparseml.image_classification.export_onnx` 
+command that you can use to load the checkpoint and create a new `model.onnx` file. 
+Be sure the `--model_path` argument points to your trained model. 
+
+```bash
+sparseml.image_classification.export_onnx \
+    --arch-key resnet50 --dataset imagenet \
+    --dataset-path ~/datasets/ILSVRC2012 \
+    --checkpoint-path ~/checkpoints/resnet50_checkpoint.pth
+```
+
+### DeepSparse Engine Deployment
+
+Once the model is exported an ONNX format, it is ready for deployment with the 
+[DeepSparse] Engine. The deployment is intuitive due to the [DeepSparse] Python API.
+
+```python
+from deepsparse import pipeline
+
+cv_pipeline = pipeline(
+  task='image_classification', 
+  model_path='zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned95-none',  # Path to checkpoint or SparseZoo stub
+)
+
+input_image = ... # Read input images
+inference = cv_pipeline(images=input_image)
+```
+To learn more, refer to the [appropriate documentation in the DeepSparse repository](https://github.com/neuralmagic/deepsparse/tree/main/examples/classification)
+
+#### Server
+We also provide a `deepsparse.server` utility for quick deployment, 
+just install [DeepSparse] using `pip install deepsparse[server]` and use the following 
+command to spin up a server hosting a model: 
+
+```bash
+deepsparse.server \
+    --task image_classification \
+    --model_path zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned95-none
+```
+This will start the server at `http://localhost:5543/predict`, now we can make 
+`POST` requests to the endpoint, a valid response will include the inference results,
+An example client is given in [Client]
+
+#### Client
+
+The following client can be used to make requests to an image classification 
+[DeepSparse] server.
+
+`imagenet_client.py`
```

Comment:
I think this script is quite useful, but you also have it in your deepsparse documentation, where I think it fits much better. I would remove it in sparseml docs, but keep a pointer to it in deepsparse docs.

Reply:
Removed!